src/openforms/forms/views/form.py

```diff
@@ -28,6 +30,10 @@ def get(self, request, *args, **kwargs):
             return HttpResponseRedirect(config.main_website)
 
 
+@method_decorator(
+    conditional_search_engine_index(config_attr="allow_indexing_form_detail"),
+    name="dispatch",
```

Comment:
no - this is the regular Django view that renders the form-detail template, which in turn includes the JS/CSS for the SDK.

This is the backend URL to a form that is typically hit by search engines, and they process the HTML meta tags/response headers to decide what to do with it.

the SDK does not have to do anything, and API endpoints are not affected by this.

If you embed a form in a CMS through the JS snippet, then it's up to the CMS to prevent indexing that page - we can't do anything about that.